pipeline-model-api/pom.xml

```diff
@@ -38,6 +38,10 @@
   <description>Model API for Declarative Pipeline</description>
   <url>https://github.com/jenkinsci/pipeline-model-definition-plugin</url>
 
+  <properties>
+    <hpi.bundledArtifacts>btf,jackson-coreutils,jackson-coreutils-equivalence,jopt-simple,json-schema-core,json-schema-validator,libphonenumber,mailapi,msg-simple,rhino,uri-template</hpi.bundledArtifacts>
```

Comment:
`jackson-coreutils` seems wrong. Not sure about `jackson-coreutils-equivalence`.

I think `libphonenumber,mailapi,msg-simple,rhino,uri-template` might all be transitive dependencies of `json-schema-validator`, but it seems kind of crazy to pick up Rhino just for validation.

No idea what `btf` is.